docs/sources/operators-guide/run-production-environment/production-tips/index.md

```diff
@@ -124,3 +124,29 @@ Running a dedicated Memcached cluster for each cache type is not required, but r
 
 We recommend securing the Grafana Mimir cluster.
 For more information about securing a Mimir cluster, refer to [Secure Grafana Mimir]({{< relref "../../secure/_index.md" >}}).
+
+## Network
+
+Most of the communication between Mimir components occurs over gRPC. The gRPC
+connection does not use any compression by default. This can be most noticeable in the connection between a
+distributor and an ingester.
```

Comment:
Reason is explained here: https://github.com/grafana/mimir/blob/main/docs/sources/operators-guide/configure/configuring-zone-aware-replication.md?plain=1#L57-L59, if you want to include it.

Reply:
I'd fall back to Ursula - is this necessary detail? It does not give any actionable information to the user. But will give an explanation if they are curious. I am not sure how curious our users are.